popup.html

```diff
@@ -37,9 +37,12 @@ <h1 class="title">Quack</h1>
         <img src="utils/copy-icon.png" id="copyPortfolio" class="copy-icon" alt="Copy">
     </div>
     <div id="custom-links"></div>
-
-    <button id="addLink" class="button-35" title="add custom link">+</button>
-    <button id="save" class="button-35" title="save links">Saves</button>
+    <div class="button-container">
+        <button id="addLink" class="button-35" title="add custom link">+</button>
+        <button id="save" class="button-35" title="save links">Save</button>
+        <button id="autoSave" class="button-35 auto-save-enabled" title="toggle auto-save">Auto-save</button>
+        <button id="settingsButton" class="button-35" title="edit custom links">Edit</button>
```

Comment:
The edit button is not intuitively easy to use. Since we're opting for speed and ease of use, I prefer the X button like previously. Links can be edited dynamically and I like the auto-save mode.